reference-apps/aaos-unbundled/media/src/main/java/com/android/designcompose/reference/media/MainActivity.kt

```diff
@@ -1,223 +0,0 @@
-/*
- * Copyright 2023 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.android.designcompose.reference.media
-
-import android.graphics.Bitmap
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
-import com.android.designcompose.DesignComposeCallbacks
-import com.android.designcompose.DesignNodeData
-import com.android.designcompose.ImageReplacementContext
-import com.android.designcompose.ListContent
-import com.android.designcompose.OpenLinkCallback
-import com.android.designcompose.ParentComponentInfo
-import com.android.designcompose.TapCallback
-import com.android.designcompose.serdegen.NodeQuery
-
-enum class PlayState {
-    Play,
-    Pause
-}
-
-enum class BrowseItemType {
-    Grid,
-    List
-}
-
-enum class CurrentlyPlaying {
-    On,
-    Off
-}
-
-enum class PageHeaderType {
-    Nav,
-    DrillDown
-}
-
-enum class SourceButtonType {
-    Selected,
-    Unselected
-}
-
-enum class NavButtonType {
-    Selected,
-    Unselected
-}
-
-interface MediaInterface {
-    @Composable
-    fun CustomActionButton(
-        modifier: Modifier = Modifier,
-        openLinkCallback: OpenLinkCallback? = null,
-        designComposeCallbacks: DesignComposeCallbacks? = null,
-        key: String? = null,
-        icon: @Composable (ImageReplacementContext) -> Bitmap?,
-        onTap: TapCallback,
-    ) {}
-
-    fun CustomActionButtonDesignNodeData(): DesignNodeData {
-        return DesignNodeData()
-    }
-
-    @Composable
-    fun BrowsePage(
```

Comment:
This function is not used anywhere. So it is not added to the CenterDisplay interface. Let me know if it makes sense.

Reply:
Yep